dpnp/backend/src/queue_sycl.cpp

```diff
@@ -142,6 +143,16 @@ void backend_sycl::backend_sycl_rng_engine_init(size_t seed)
     rng_engine = new mkl_rng::mt19937(DPNP_QUEUE, seed);
 }
 
+void backend_sycl::backend_sycl_rng_stream_init(size_t seed)
```

Comment:
Do we really need this stream?
Can we avoid keep it between function calls and create it every time in the function that used it?

Reply:
Yes, we really need it. It is needed for saving prev. state and reseeding.
The only thing that is needed is rename `backend_sycl` to something like `backend_base`, that contains all these states and queue.